src/generic/network-options.ts

```diff
@@ -57,6 +57,7 @@ export var NETWORK_OPTIONS :{[name:string]:social.NetworkOptions} = {
     isEncrypted: true
   },
   'Cloud': {
+    displayName: 'Create a private cloud server',
```

Comment:
This change allows the cloud option in the networks list to show this label, but it also changes it in other places (such as browser notifications) too, e.g. when you delete a cloud server, you see "You have been logged out of Create a private cloud server". Would it be best to not use the displayName field for this, and have special case logic to change the label in the networks list just for Cloud?

Should there also be special case logic to change the notification when you delete a cloud server to say "Your cloud server (Singapore - 123.45.67.89) has been deleted" instead of "you have been logged out of cloud"?